core/server/common/src/main/java/alluxio/master/journal/raft/transport/CopycatGrpcServer.java

```diff
@@ -97,8 +99,8 @@ public CopycatGrpcServer(AlluxioConfiguration conf, UserState userState,
 
       // Create gRPC server.
       mGrpcServer = GrpcServerBuilder
-          .forAddress(GrpcServerAddress.create(address.host(), address.socketAddress()), mConf,
-              mUserState)
+          .forAddress(GrpcServerAddress.create(address.host(),
+              new InetSocketAddress(address.host(), address.port())), mConf, mUserState)
```

Comment:
@gpang I reverted the separate bind address for raft and passing the resolved master address to bind